torch/distributed/algorithms/quantization/quantization.py

```diff
@@ -86,13 +99,16 @@ def auto_quantize(func, qtype, quant_loss=None):
     """
     This is a prototype API that automatically quantize the input tensors, choose the precision types, and
     pass other necessary arguments and then dequantizes the output.
+
     Currently it only supports:
         . FP16 quantization method
```

Comment:
nit: update the doc to reflect we added the support for bfp16, also make a note about bfp16 limitation (only supports 2d tensor for now)